src/query.js

```diff
@@ -0,0 +1,87 @@
+import prettyjson from 'prettyjson';
+import nmo from './nmo.js';
+import Promise from 'bluebird';
+import { getUrlFromCluster, sendJsonToNode } from './utils';
+
+export function cli (cluster, dbname, ...args) {
+  return new Promise((resolve, reject) => {
+    if (!cluster || !dbname || args.length === 0) {
+      const msg = [
+        'Usage:',
+        '',
+        'nmo query <cluster> <database> create <fields> [--json]',
+        'nmo query <cluster> <database> <selector> [--json]',
+      ].join('\n');
+
+      const err = new Error(msg);
+      err.type = 'EUSAGE';
+      return reject(err);
+    }
+
+    let promise;
+
+    if (args[0] === 'create') {
+      if (!args[1] || !args[1].length === 0) {
+        const err = new Error('Please supply one or more fields to be indexed');
+        err.type = 'EUSAGE';
+        reject(err);
+        return;
+      }
+
+      const fields = args[1].split(',').map(f => f.trim());
```

Comment:
maybe we can test for `/^}/.test(args0)` and then display some help to the user (something like:

err.type = EUSAGE;
err.message = Looks like you have forgotten to wrap the query in single quotesplease include quotes. Example usage: `nmo query anemone restaurants '{"selector": {"id": {"$gt":null}}}'`